### PR TITLE
Connecting teamspage to backend

### PR DESCRIPTION
**Pull request by katherinj**
Connected TeamsPage to backend. Added params to user + team route to be able to verify a user is valid when creating a new team and adding members to that team. Displayed teams a user is apart of, as well as the members of a specific team that has been clicked on. Added error checking to make sure Name input has been filled out before creating a team, as well as error checking to make sure a user exists with the email a user is trying to add to the team.  Added additional comments to the TeamsPage components.

### CODE REVIEW

File: bugtracker-api/routes/team.js [hunk 1]
```diff
@@ -84,14 +82,13 @@ router.get("/:teamId", security.requireAuthenticatedUser, async(req,res,next) =>
 
 //FUNCTION TO ADD A NEW MEMBER TO A TEAM
 router.patch("/:teamId/add", security.requireAuthenticatedUser, async(req,res,next) => {
+    console.log("Team trying router: ", req.body)
```
Comment (LifeBringer) [inline bugtracker-api/routes/team.js:85]: Remove the console.log

**Review by LifeBringer — CHANGES_REQUESTED** [review]
Rogue console.log

File: bugtracker-api/routes/team.js (hunk 1 above)
Comment (Rebecca-CYLi) [reply bugtracker-api/routes/team.js]: cool catch, resolved

**Review by Rebecca-CYLi — APPROVED** [review]
Fixed console logs, LGTM

**Pull request merged**